app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -637,13 +690,20 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
           {chooseWalletModal}
         </View>
         <View style={Styles.fieldContainer}>
-          <Text style={Styles.fieldTitleText}>{LL.SendBitcoinScreen.amount()}</Text>
+          <View style={Styles.amountHeader}>
```

Comment:
One thing to keep in mind is that the `Send Max` functionality isn't universally available right now. Currently it can only happen with onchain spends from your btc wallet (using the OnChainPaymentSendAllMutation), as well as intraledger payments with their normal mutation because they have no fee. It will not work with lightning payments right now as there is no mutation for it yet.

Reply:
Indeed. I haven't fully digested how `send-max` works, so this is a good to note! One thing that may make this "universal" is to compare the final send on the confirm UI with the total wallet balance. If they are the same, then `max-send` is assumed. And the final send would become the total send (minus fee). As such, even if the BE doesn't have it for all payment types it can still be "faked" for all payment types. I am not sure if this is desirable, and was actually writing up some thoughts in the chat as you commented. Will post those soon. But i def need to do more research on how all the piping works for max-send.